backend/question-service/utils/database.ts

```diff
@@ -1,8 +1,4 @@
 import mongoose from "mongoose";
-import dotenv from "dotenv";
-import path from "path";
-
-dotenv.config({ path: path.resolve(__dirname, '../.env') });
```

Comment:
We need this line, and the .env file this is referencing is still in the project so this line is ok, and needed for the process.env call below this. This isn't the line I wanted you to remove. Remove the one in app.ts, leave this one in.

Reply:
Removed, the one in database also dont need, as it is already been declared in the app.ts, as it has the dependency in the app.ts